core/src/main/scala/org/apache/pekko/persistence/jdbc/journal/dao/JournalQueries.scala

```diff
@@ -50,49 +54,39 @@ class JournalQueries(
     }
   }
 
-  private def selectAllJournalForPersistenceIdDesc(persistenceId: Rep[String]) =
-    selectAllJournalForPersistenceId(persistenceId).sortBy(_.sequenceNumber.desc)
-
-  private def selectAllJournalForPersistenceId(persistenceId: Rep[String]) =
-    JournalTable.filter(_.persistenceId === persistenceId).sortBy(_.sequenceNumber.desc)
-
   def delete(persistenceId: String, toSequenceNr: Long) = {
-    JournalTable.filter(_.persistenceId === persistenceId).filter(_.sequenceNumber <= toSequenceNr).delete
+    JournalTable
+      .filter(_.persistenceId === persistenceId)
+      .filter(_.sequenceNumber <= toSequenceNr)
+      .delete
   }
 
-  def markJournalMessagesAsDeleted(persistenceId: String, maxSequenceNr: Long) =
+  def markMaxSequenceNrJournalMessagesAsDeleted(persistenceId: String, maxSequenceNr: Long) =
```

Comment:
maybe rename `markSequenceNrJournalMessagesAsDeleted(persistenceId: String, sequenceNr: Long)`
since it marks the given sequence number as deleted

Reply:
What about this?  remove the plural form in the method name, as we only updated one row of the journal.

```diff
 -  def markJournalMessagesAsDeleted(persistenceId: String, maxSequenceNr: Long) =
 +  def markJournalMessageAsDeleted(persistenceId: String, sequenceNr: Long) =
```

Reply:
NVM, I rename it as `markSeqNrJournalMessagesAsDeleted ` which is shorter of`markSequenceNrJournalMessagesAsDeleted `

Reply:
right, better 👍